modules/ROOT/pages/mergetags.adoc

```diff
@@ -101,17 +101,19 @@ Here is an example of the {pluginname} HTML structure.
 </span>
 ----
 
-== Merge Tags and the Insertion Point Marker
+== Merge Tags and the Advanced Templates Insertion Point Marker
 
 include::partial$misc/admon-requires-6.7v.adoc[]
 
 The xref:advanced-templates.adoc[Advanced Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
 
-Even though this fixed string uses the same default delimiting characters as individual merge tags, the {pluginname} plugin knows to ignore this particular string.
+Even though this fixed string uses the same default delimiting characters as individual merge tags, this fixed string does not interfere or otherwise interact with any {pluginname} configuration.
 
-Consequently, `+{{mce-cursor}}+`, is not supported as an individual merge tag.
+The Advanced Templates plugin replaces the Insertion Point Marker string with a hook element before inserting a template containing the string in to a {productname} instance.
```

Comment:
I'm not sure whether we should share those implementation details as they could be changed. Maybe skip the mention of this feature in MergeTag plugin docs as long as nothing was done here. They look similar but unrelated to each other. Merge Tags are part of the content though a point marker is just a tool to indicate the location of caret and can be used only in template content.

Reply:
I’ve edited this to remove mention of the hook element.

I’m arguing for leaving this section in place in `mergetags.adoc`, however. Using the same default delimiting characters automatically raises the question _is this string related to merge tags?_.

Better to answer the question before it is asked.